few minor typos corrected

rksin8 opened the pull request:
Few minor typos corrected

rksin8 committed:
few minor type corrected

serine commented on book/05-more-rmarkdown.Rmd:
````diff
@@ -164,8 +164,7 @@ Firstly lets make sure we have our data properly filtered. We are going to filte
 text <- c("```{r}",
-          "df2 <- df %>% filter(Airline == 'Jetstar' | Airline == 'Qantas')",
-	  "                     Route != 'All Ports-All Ports')",
+          "df2 <- df %>% filter( (Airline == 'Jetstar' | Airline == 'Qantas') & Route != 'All Ports-All Ports')",
````
Thanks for picking this up, it was a bug. I'm a little unsure if it meant to be AND (`&`) or simply comma? In this case I'm doing two separate filtering, one for airlines and one for route. I think what you have there is fine, but just a little unsure

rksin8 replied:
Modified the filter syntax, replaced & with ",".

serine commented:
@rksin8 thanks for the PR this is awesome. Can you please reset you last commit and drop `book/libs` from it. And perhaps add that to `.gitignore` those files aren't meant to be part of the repo. They are transient file made during build. My fault for not including them into `.gitignore` earlier. 

you should be able to 

```git reset HEAD~1``` this will undo last commit (I'm assuming that `book/libs`) were part of the last commit. And then just commit everything else. Don't forget to grab your commit message so that you don't have to re-type it again with

```git --no-pager log --format=%B -n1```

Cheers!

rksin8 committed:
few minor typos corrected

rksin8 committed:
Revert "few minor typos corrected"

This reverts commit d0faa87d632a083fd5c1df3d5ac503374574f7b0.

rksin8 committed:
06_yaml_string_format corrected

rksin8 committed:
Remove duplicted director

rksin8 committed:
update filter syntax in chap05

rksin8 commented:
> @rksin8 thanks for the PR this is awesome. Can you please reset you last commit and drop `book/libs` from it. And perhaps add that to `.gitignore` those files aren't meant to be part of the repo. They are transient file made during build. My fault for not including them into `.gitignore` earlier.
> 
> you should be able to
> 
> `git reset HEAD~1` this will undo last commit (I'm assuming that `book/libs`) were part of the last commit. And then just commit everything else. Don't forget to grab your commit message so that you don't have to re-type it again with
> 
> `git --no-pager log --format=%B -n1`
> 
> Cheers!

Updated.gitignore, also updated few more syntax observed during today workshop.

serine commented:
@rksin8 thanks so much for contributing! I haven't pulled this PR in time and it has gone out of sync. I made a fair few improvements and changes to the content. I'll be sure to pull any future PR promptly
